mapbox/libjava-services/src/main/java/com/mapbox/services/api/geocoding/v5/models/CarmenFeature.java

```diff
@@ -129,6 +134,18 @@ public void setRelevance(double relevance) {
   }
 
   /**
+   * Create a CarmenFeature object from JSON.
+   *
+   * @param json String of JSON making up a carmen feature.
+   * @return Carmen Feature
+   * @since 2.0.0
+   */
+  public static CarmenFeature fromJson(String json) {
+    GsonBuilder gson = new GsonBuilder();
+    return gson.create().fromJson(json, CarmenFeature.class);
```

Comment:
@cammace Don't you need to attach our custom deserializer here `gson.registerTypeAdapter(Position.class, new PositionDeserializer());`?